tests/drone/getFilesChanged.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+CHANGED_UNIT_TESTS_ONLY=True
+CHANGED_DOCS_ONLY=True
+CHANGED_UNIT_TESTS_AND_DOCS_ONLY=True
+
+DIFFS=$(git diff origin/master --name-only)
+DOCS_ONLY_CHANGES=$(echo "$DIFFS" | grep -E '(docs/.*|changelog/.*)')
+UNIT_TESTS_CHANGES=$(echo "$DIFFS" | grep 'packages/.*/tests/.*')
+DOCS_AND_UNIT_TESTS_ONLY_CHANGES=$(echo "$DIFFS" | grep -E '(docs/.*|changelog/.*|packages/.*/tests/.*)')
```

Comment:
will these regex hold if the path falls inside the docs folder?
for eg. if a file is changed with path `docs/.../packages/files/tests/unitTest.md` then you will get this in both docs only changes and unit tests changes but in reality only docs has been changed.